pkg/intervals/intervals.go

```diff
@@ -0,0 +1,281 @@
+package intervals
```

Comment:
Here and in other files let's add the license comment...

Reply:
applied